src/components/homepage/HeroSection.jsx

```diff
@@ -0,0 +1,41 @@
+import image from "../../assets/fuse machines.jpeg";
+const HeroSection = () => {
+  return (
+    <section className="bg-[#0A192F] px-6 py-6">
```

Comment:
**you can define the colors like primary , secondary colors in tailwind for reuse.**